build.sbt

```diff
@@ -11,14 +11,14 @@ val cliPack =
   taskKey[File]("Pack the CLI for the current configuration")
 
 inThisBuild(Def.settings(
-  version := "1.0.1-SNAPSHOT",
+  version := "1.5.0-SNAPSHOT",
   organization := "org.scala-js",
 
-  crossScalaVersions := Seq("2.12.10", "2.11.12", "2.13.1"),
+  crossScalaVersions := Seq("2.12.13", "2.11.12", "2.13.5"),
   scalaVersion := crossScalaVersions.value.head,
-  scalacOptions ++= Seq("-deprecation", "-feature", "-Xfatal-warnings"),
+  scalacOptions ++= Seq("-deprecation", "-feature"/*, "-Xfatal-warnings" FIMXE: commented because using legacy API */),
```

Comment:
We need a better solution for this :) Have a look here (not pretty, but the best way we could find):

https://github.com/scala-js/scala-js/blob/d09ce384d35e3310272e8099dda5f2d3fee8dac1/sbt-plugin/src/main/scala/org/scalajs/sbtplugin/ScalaJSPluginInternal.scala#L265-L292